x/leveragelp/keeper/msg_server_open.go

```diff
@@ -58,7 +58,6 @@ func (k Keeper) Open(ctx sdk.Context, msg *types.MsgOpen) (*types.MsgOpenRespons
 		sdk.NewAttribute("id", strconv.FormatInt(int64(position.Id), 10)),
 		sdk.NewAttribute("address", position.Address),
 		sdk.NewAttribute("collateral", position.Collateral.String()),
-		sdk.NewAttribute("leverage", position.Leverage.String()),
```

Comment:
maybe we can add current leverage for whole position

Reply:
@cryptokage1996 I think what @amityadav0 meant is to set leverage attribute to the dynamically calculated leverage value

Reply:
@cryptokage1996 instead of storing it to optimize data usage, can we just call the function to calculate it and provide it to the leverage attribute?